edu.gemini.lch.model/src/main/java/edu/gemini/lch/model/Interval.scala

```diff
@@ -0,0 +1,28 @@
+package edu.gemini.lch.model
+
+import java.time.Instant
+
+/**
+ * An interval is a period of time between two instants.
+ * Includes start, excludes end.
+ * Requirement: end >= start.
+ * They have fixed ms duration, i.e. end - start and are not comparable.
+ * To compare intervals, you should compare their lengths, i.e. Durations.
+ */
+case class Interval(start: Instant, end: Instant) {
```

Comment:
Typically in these cases we make the constructor private and add a companion class method for doing the construction.  The constructor there should guarantee the prerequisite that `end >= start` and then you can add a sanity check assertion.

Reply:
👍